docs/features/analysis.md

```diff
@@ -425,6 +425,48 @@ spec:
           valueFrom:
             podTemplateHashValue: Latest
 ```
+Analysis arguments also support valueFrom for reading metadata fields and passing them as arguments to AnalysisTemplate.
+An example would be to reference metadata labels like env and region and passing them along to AnalysisTemplate.
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: guestbook
+  env: test
+  region: us-west-2
```

Comment:
These need to go under `metadata.labels` not directly under metadata

Reply:
@jessesuen - you are right , this was a typo , i have resolved it.